README.md

```diff
@@ -260,7 +260,7 @@ Here is the BibTeX entry:
 ```bibtex
 @misc{wood2022kerascv,
```

Comment:
I think we should probably plan to change this to eventually point to the citation of the paper we're writing, and for now let's just make an authors/maintainers list and include you there instead.

That way, folks can also easily see who to reach out to with questions as well

Reply:
That sounds good!